scripts/core/ui/components/PlainTextEditor/PlainTextEditor.tsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import {
+    Editor,
+    EditorState,
+    ContentState,
+    Modifier,
+    SelectionState,
+    CompositeDecorator,
+} from 'draft-js';
+import './styles.scss';
+import {getSpellchecker} from 'core/editor3/components/spellchecker/default-spellcheckers';
+import {
+    getSpellcheckWarningsByBlock,
+    getSpellcheckingDecorator,
+} from 'core/editor3/components/spellchecker/SpellcheckerDecorator';
+import {getDraftSelectionForEntireContent} from 'core/editor3/helpers/getDraftSelectionForEntireContent';
+
+export interface IProps {
+    value: string;
+    onChange: (newValue: string, props: IProps) => void;
```

Comment:
oops, forgot about that one. Yes, it is mandatory, when we handle editor changes we usually trigger an `onchange` function in angular, and that function usually requires data like the field you're changing ([example](https://github.com/superdesk/superdesk-client-core/blob/59c87dcb54878baa906d3b5fe6bc9f6cc36382f0/scripts/apps/authoring/media/views/media-metadata-editor-directive.html#L33)). It also makes the editor more flexible as it can return any data you pass to it.